src/main/routes/createDefinition.ts

```diff
@@ -22,7 +22,7 @@ router.get("/createdefinition", (req, res, next) => {
     responseContent.heading = "Create Definition";
     responseContent.submitButtonText = "Create";
     if (req.session.error) {
-      responseContent.error = sanitize(req.session.error);
+      responseContent.error = JSON.parse(sanitize(JSON.stringify(req.session.error)));
```

Comment:
sanitize required for output data as well?

Reply:
I recall this was how @andrewfolga fixed another issue before as part of ITHC pentest remediations.